app/profile.html

```diff
@@ -26,35 +26,74 @@
       <!-- Customized libraries shared by pages -->
       <script src="js/site.js"></script>
 	  	  
+
+      <style>
+      .ng-invalid.ng-dirty {
+      border-color: #FA787E;
+      }
+        
+      .ng-valid.ng-dirty {
+      border-color: #78FA89;
+      }
+      </style>
+
+    
    </head>
    <body>
        
       <!--navbar-->
       <div ng-include="'navbar.html'" id='navbarMenu'></div>
-   
-       <h1>Profile</h1>
-       <div class="row">
-           <div class="col-md-12">
-               <ul>
-                    <li>Name:<input type="text" name="userName" ng-model="userData.name"></li>
-                    <li>Language:<input type="text" name="lan" ng-model="txtLanguage"><button class="btn btn-primary" ng-click="addLanguage()">Add</button></li>
-                    <li>Gpa:<input type="number" name="gpa" ng-model="userData.gpa"></li>
-                    <li>Description:</li>
-                    <textarea ng-model="userData.description"></textarea>
-                    <li>Language Added:</li>
-                    <ol>
-                        <div ng-repeat="lan in userData.language">
-                            <li>{{lan}}<button class="btn btn-primary btn-sm" ng-click="removeLanguage(lan)">Remove</button></li>
-                        </div>
-                    </ol>
-                    <li>CurrentTeam:{{userData.team}}</li>
-                    <li>Eventlist:{{userData.eventlist}}</li>
-                    <li>My User id: {{userData.userid}}</li>
-               </ul>
-               <!--button class="btn btn-primary" ng-click="testeventlist()">test event list</button-->
-               <button class="btn btn-primary" ng-click="submitUserData()">Submit</button>
-            </div>
-       </div>
+      
+      <div class="container">
+        <div class="page-header">
+  			<h1>Profile</h1>
+		</div>
+
+        <div class="row">
+            <form name="ProfileForm" class="css-form col-md-6" ng-submit="ProfileForm.$valid && submitUserData()" novalidate>
```

Comment:
Remove ng-submit here and add ng-click='submitUserData()' in line 76 should solve the problem